pom.xml

```diff
@@ -79,7 +91,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>edge-common</artifactId>
-      <version>2.0.2</version>
+      <version>4.2.0-SNAPSHOT</version>
```

Comment:
Why was a `snapshot` dependency chosen?

I think version {{4.1.0}} should be sufficient for this work, as that is the latest release and support vert.x 4.x

FOLIO modules should avoid using snapshot dependencies unless absolutely necessary for a change, as it can lead to unstable builds and blocks the module from being released.

Reply:
This module relies on the HTTP client provided by edge-common timing out requests and issuing a specific exception when doing so.

Changes were made in version 4.1.0 that meant this no longer happened. 

This has hopefully been resolved, please try using version 4.2.0 to check if this resolves the issue.